tripal_chado/includes/tripal_chado.fields.inc

```diff
@@ -2813,15 +2813,19 @@ function tripal_chado_bundle_get_properties($table_name, $prop_table, $type_tabl
     $args[':cvterm_id'] = $cvterm_id;
     $props = chado_query($sql, $args);
   }
-  
+
   // Iterate through all of the properties and do some final checks to see
   // which ones should be added.
   $prop_arr = [];
+  if($props === null) {
```

Comment:
I've added this here because I kept getting the following error:

```shell
WD php: Error: Call to a member function fetchObject() on null in tripal_chado_bundle_get_properties() (line 2820 of                          [error]
/Users/Almsaeed/Work/DevSites/Tripal/sites/all/modules/tripal/tripal_chado/includes/tripal_chado.fields.inc).
Cannot modify header information - headers already sent by (output started at                                                                 [warning]
/Users/Almsaeed/Work/DevSites/Tripal/sites/all/modules/tripal/tripal/api/tripal.jobs.api.inc:388) bootstrap.inc:1486
Error: Call to a member function fetchObject() on null in tripal_chado_bundle_get_properties() (line 2820 of /Users/Almsaeed/Work/DevSites/Tripal/sites/all/modules/tripal/tripal_chado/includes/tripal_chado.fields.inc).
Drush command terminated abnormally due to an unrecoverable error.                                                                            [error]
```